Form-Controls/index.html

```diff
@@ -1,28 +1,120 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1 class="title">Product Pick</h1>
+    </header>
+    <main class="main">
+      <form class="form" action="#" method="get" enctype="multipart/form-data">
+        <div class="shop">
+          <p>First Name:</p>
+          <label for="first_name"></label>
+          <input
+            type="text"
+            name="first_name"
+            id="first_name"
+            placeholder="Enter First Name"
+            value=""
+            required
+          />
+        </div>
+
+        <div class="shop">
+          <p>Last Name:</p>
```

Comment:
```suggestion
          <label for="last_name">Last Name:</label>
```

The label is labelling the input. In devtools, open the Accessibility tree and add some text to your empty label. Can you see the difference?



https://developer.chrome.com/docs/devtools/accessibility/reference#explore-tree